libs/gltfio/include/gltfio/AssetLoader.h

```diff
@@ -62,6 +74,9 @@ struct AssetConfiguration {
 
     //! Optional default node name for anonymous nodes
     char* defaultNodeName = nullptr;
+
+    //! Optional to enable mikktspace tangents.
+    AssetConfigurationExtended* ext = nullptr;
```

Comment:
Maybe comment on the required lifetime for this pointer? Based on the code it seems like it only needs to exist during `AssetLoader::create`

Reply:
Good point.  Comment added.